### PR TITLE
Create ci.yml

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

**Comment by schillic** [comment]
To add `NeuralVerification`, we could try to add a Julia command as given [here](https://github.com/invenia/PkgTemplates.jl/blob/88b55242d47fb91541bee4c338ff028493ddbc78/test/fixtures/DocumenterGitHubActions/.github/workflows/CI.yml#L46-L50).

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: "Set up Julia"
+        uses: julia-actions/setup-julia@v1
+        with:
+          version: ${{ matrix.julia-version }}
+          arch: ${{ matrix.julia-arch }}
+      - run: |
+          julia --project=docs -e '
```
Comment (mforets) [inline .github/workflows/ci.yml:34]:
```suggestion
          julia --project -e '
```

**Comment by mforets** [comment]
the previous commit had worked already, but there was a  parsing problem in the windows build. now i'm trying to write all the statements in the  same line to see if that fixes the windows build.

**Comment by mforets** [comment]
it didn't work:

> Run julia --project -e 'using Pkg; Pkg.develop(PackageSpec(name="NeuralVerification", url="https://github.com/sisl/NeuralVerification.jl")); Pkg.instantiate()'
ERROR: syntax: "//" is not a unary operator

merging ..

**Comment by schillic** [comment]
Should we then deactivate the Windows build?

**Pull request merged**